CHANGELOG.rst

```diff
@@ -71,6 +70,8 @@
   Fixes a pytest internal error for dynamically generated code (fixtures and tests)
   where source lines are fake by intention.
 
+* Add ability to add global properties in the final xunit output file.
```

Comment:
This should go into the `2.10.0` section, in the `features` branch. `2.9.0` has been released already, and `2.9.X` may contain only bug fixes.